src/screens/Customers/CustomerProfile.tsx

```diff
@@ -41,14 +41,20 @@ const styles = (theme: Theme) =>
     meterInfoGrid: {
       display: 'flex',
     },
-    meterInfoCol: {
+    leftMeterInfoCol: {
       width: '50%',
       padding: '5px',
+      paddingLeft: '0px',
+    },
+    rightMeterInfoCol: {
+      width: '50%',
+      padding: '5px',
+      paddingRight: '0px',
```

Comment:
Nit: instead of setting padding then overriding paddingRight, it looks cleaner if you just set it all in one line using this shorthand: `padding: top right bottom left` (see https://www.w3schools.com/css/css_padding.asp)